resolwe_bio/processes/reads_processing/bbduk.yml

```diff
@@ -0,0 +1,655 @@
+# =============================================
+# Processors for reads procesing - BBduk
+# =============================================
+#
+# Processors for NGS reads filtering/trimming
+---
+
+- slug: bbduk-single
+  name: BBDuk from BBMap 36.20 (single-end)
+  requirements:
+    expression-engine: jinja
+  data_name: BBduk'{{ reads.fastq.0.file|default("?") }}'
```

Comment:
This will create a data name that will be without whitespace between "BBduk" and Read's name.